tests/functional/test_freeze.py

```diff
@@ -258,41 +259,31 @@ def test_freeze_mercurial_clone(script, tmpdir):
     _check_output(result, expected)
 
 
-@pytest.mark.network
 def test_freeze_bazaar_clone(script, tmpdir):
     """
     Test freezing a Bazaar clone.
 
     """
-
-    checkout_path = local_checkout(
-        'bzr+http://bazaar.launchpad.net/%7Edjango-wikiapp/django-wikiapp/'
-        'release-0.1',
-        tmpdir.join("cache"),
-    )
-    # bzr internally stores windows drives as uppercase; we'll match that.
+    try:
+        checkout_path = _create_test_package(script, vcs='bazaar')
+    except OSError as e:
+        pytest.skip('Invoking `bzr` failed: %s' % e)
```

Comment:
OK, I changed it from `pytest.skip` to `pytest.fail` and here's how it fails when `bzr` is not installed:

```python
$ .tox/py27/bin/py.test tests/functional/test_freeze.py -k test_freeze_bazaar_clone --tb=short --pdb
Test session starts (platform: darwin, Python 2.7.9, pytest 2.6.4, pytest-sugar 0.3.6)
plugins: capturelog, cov, sugar, timeout, xdist


――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― test_freeze_bazaar_clone ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
tests/functional/test_freeze.py:270: in test_freeze_bazaar_clone
    pytest.fail('Invoking `bzr` failed: %s' % e)
E   Failed: Invoking `bzr` failed: [Errno 2] No such file or directory


>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> traceback >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
tests/functional/test_freeze.py:270: in test_freeze_bazaar_clone
    pytest.fail('Invoking `bzr` failed: %s' % e)
E   Failed: Invoking `bzr` failed: [Errno 2] No such file or directory
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> entering PDB >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
[13] > /Users/marca/dev/git-repos/pip/tests/functional/test_freeze.py(270)test_freeze_bazaar_clone()
-> pytest.fail('Invoking `bzr` failed: %s' % e)
   1 frame hidden (try 'help hidden_frames')
```